scripts/check_total_coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+coverage_dir=$(PWD)/coverage
+combined_coverage_file=$coverage_dir/combined.txt
+COVERAGE_THRESHOLD=15.0 # percentage threshold of code coverage required
+
+echo "Generating coverage report in text format..."
+included_packages=$(go list ./... | grep -v /tests/ | grep -v '/sdk/') # not including 'tests' and 'sdk'
+go tool covdata merge -i=$coverage_dir/e2e,$coverage_dir/ut -o=$coverage_dir -pkg=${included_packages//$'\n'/,}
```

Comment:
do we need th pkg param? why is that?

Reply:
this param tells golang coverage tool about which pkg to analyze coverage. without this param the coverage gave an empty report

Reply:
I believe I used merged without the pkg param and it just worked